uint8 status
Header header

segway_sim/state cur_state
segway_sim/input cur_input